src/vs/workbench/browser/parts/menubar/media/menubarpart.css

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.monaco-workbench > .part.menubar {
+	display: flex;
+	position: absolute;
+	order: 3;
+	box-sizing: border-box;
+	margin-left: 35px;
+}
+
+.monaco-workbench.fullscreen > .part.menubar {
+	position: absolute;
+	width: 100%;
+	margin: 0px;
+	padding: 0px 5px;
+}
+
+.monaco-workbench > .part.menubar > .menubar-menu-button {
+	display: flex;
+	align-items: center;
+	min-width: 0;
+	box-sizing: border-box;
+	padding: 0px 5px;
+	position: relative;
+	z-index: 10;
+	cursor: pointer;
+	-webkit-app-region: no-drag;
+}
+
+.monaco-workbench > .part.menubar > .menubar-menu-button.open,
+.monaco-workbench > .part.menubar > .menubar-menu-button:hover {
+	background-color: rgba(255, 255, 255, 0.1);
```

Comment:
@sbatten any color defined in CSS needs to be externalized to a themable color in [`theme.ts`](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/common/theme.ts#L1) so that theme authors can tweak them.